chore: mention formatter and disassember in README.md (#2824)
tact-lang/tact: README.md

- [Misti](https://github.com/nowarp/misti) - Static smart contract analyzer.
- [TON Symbolic Analyzer (TSA)](https://github.com/espritoxyz/tsa) - Static smart contract analysis tool based on symbolic execution.

###### Utility

> - Formatter (`tact-fmt`) — Idiomatic and official formatter. It ships with the Tact Language Server, VS Code extension, and as a standalone binary with the compiler. You can invoke it by running `npx tact-fmt` in your Tact projects.

Reviewer: ```suggestion
- Formatter (`tact-fmt`) — The official formatter. It ships with the Tact Language Server, VS Code extension, and as a standalone binary with the compiler. You can invoke it by running `npx tact-fmt` in your Tact projects.
```

> - BoC Disassembly (`unboc`) — Disassembly of `.boc` files from the [@tact-lang/opcode](https://npmjs.com/package/@tact-lang/opcode) package, which ships as a standalone binary with the compiler. You can invoke it by running `npx unboc` in your Tact projects.

Reviewer: ```suggestion
- BoC Disassembly (`unboc`) — Disassembler for `.boc` files. Ships as a standalone binary with the compiler. You can invoke it by running `npx unboc` in your Tact projects.
```

Reviewer: the notion of BoC is highly specific to TON blockchain, so a link with explanation is needed

- [Misti](https://github.com/nowarp/misti) - Static smart contract analyzer.
- [TON Symbolic Analyzer (TSA)](https://github.com/espritoxyz/tsa) - Static smart contract analysis tool based on symbolic execution.

###### Utility

- Formatter (`tact-fmt`) — Idiomatic and official formatter. It ships with the Tact Language Server, VS Code extension, and as a standalone binary with the compiler. You can invoke it by running `npx tact-fmt` in your Tact projects.

> - BoC Disassembler (`unboc`) — Disassembler of `.boc` files from the [@tact-lang/opcode](https://npmjs.com/package/@tact-lang/opcode) package, which ships as a standalone binary with the compiler. You can invoke it by running `npx unboc` in your Tact projects.

Reviewer 2: ```suggestion
- BoC Disassembler (`unboc`) — Disassembler for [`.boc`](https://docs.tact-lang.org/book/cells/#cells-boc) files. Ships as a standalone binary with the compiler. You can invoke it by running `npx unboc` in your Tact projects.
```
